gorm.go

```diff
@@ -24,6 +24,12 @@ type Config struct {
 	SkipDefaultTransaction    bool
 	DefaultTransactionTimeout time.Duration
 
+	// Not all database support LastInsertId, you can set `IgnoreLastInsertIDWhenNotSupport` to true in those cases
+	IgnoreLastInsertIDWhenNotSupport bool
```

Comment:
I think users can config it themselves, even if the driver supports it, it is not necessary to return `LastInsertID`